core/src/main/java/tachyon/worker/WorkerStorage.java

```diff
@@ -322,6 +324,12 @@ public WorkerStorage(InetSocketAddress masterAddress, ExecutorService executorSe
 
     mDataFolder = WorkerConf.get().DATA_FOLDER;
     mUserFolder = CommonUtils.concat(mDataFolder, WorkerConf.USER_TEMP_RELATIVE_FOLDER);
+
+    if (WorkerConf.get().EVICT_STRATEGY_TYPE.needReferenceFrequency()) {
+      mLastAttenuateBlockReferenceMs = System.nanoTime() / 1000000L;
```

Comment:
Would be better to use the [TimeUnit](http://docs.oracle.com/javase/7/docs/api/java/util/concurrent/TimeUnit.html) API and do a `TimeUnit.NANOSECONDS.toMillis(System.nanoTime())`